### PR TITLE
Authentication Basics && Security Configuration && Project: Members Only: Add descriptive link texts

### PR DESCRIPTION
**Pull request by looonnng**
## Because

Update link text to improve accessibility

## This PR

- Replace old link texts
- Add blank lines to div

## Issue

Related to #27819

## Pull Request Requirements

-   [X] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [X] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [X] The `Because` section summarizes the reason for this PR
-   [X] The `This PR` section has a bullet point list describing the changes in this PR
-   [X] If this PR addresses an open issue, it is linked in the `Issue` section
-   [X] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [X] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
There we go, things are much more up to date! :tada:
Would you be able to sort out the linting errors here as well please? The errors should be somewhat self explanatory.

It also looks like you may have some sort of auto-formatter perhaps? As the 3 files' headings' levels have had some changed from level 3s to 1s and 2s. These all need to be reverted to level 3s.

The repo contributing guide has instructions on running lint checks locally, and autofixing some errors (some don't have autofixes so will still need manual fixing).

**Comment by looonnng** [comment]
Hi Mao, sorry for the monstrosity of my commits :O
Is there anything I could do in the future to make this cleaner?
Thanks again for the opportunity, I've learned a lot from contributing!

File: nodeJS/authentication/project_members_only.md
Comment (MaoShizhong) [inline nodeJS/authentication/project_members_only.md:20]:
```suggestion
1. When you're satisfied with your work, deploy your project on your chosen PaaS ([list of PaaS providers from the Deployment lesson](https://www.theodinproject.com/lessons/nodejs-deployment)) and share it below!
```
Think you just forgot this one when doing the fresh PR :P

**Review by MaoShizhong — CHANGES_REQUESTED** [review]

**Comment by MaoShizhong** [comment]
@looonnng Don't worry about the commits thing that much. We squash commits before merging anyway, and typically a bit of a mess isn't that big of a deal to review with the review tools. Now if you were doing really weird scattered unrelated commits without much rhyme or reason to them, and force-pushing left right and centre, now that might make things a little confusing depending on the situation XD But I don't think you'd do that :P 

In terms of the workflow for future PRs, just make sure you keep your local fork's main branch up to date with upstream main before making a new branch off it (and make sure you make new branches off main as opposed to off a previous branch XD).
The contributing guides (mainly the curriculum repo one) have all the instructions for workflow in general, as well as local linting. So rather than fixing some, pushing commits then being hit with a bunch of errors via the action, you can run the linter locally. Multiple separate commits are completely fine, by the way. Just you may prefer not pushing them until the errors are gone, or you can't fix any further and need further instruction/clarification :) Saves you having to get a bunch of "failed action run" notifications.

**Comment by looonnng** [comment]
Silly me, I've updated the old link text as suggested, let me know if anything else popped up.

Thank you so much again!

**Review by MaoShizhong — APPROVED** [review]
With that, we should be good to merge! If anything was missed, that'll be on me :stuck_out_tongue_closed_eyes:

**Pull request merged**
